source/extensions/filters/http/ext_proc/processor_state.cc

```diff
@@ -249,7 +258,7 @@ absl::Status ProcessorState::handleBodyResponse(const BodyResponse& response) {
         }
       }
     } else {
-      onFinishProcessorCall();
+      onFinishProcessorCall(Grpc::Status::FailedPrecondition);
```

Comment:
I wonder if it'll be confusing to inject these fake (I think?) grpc response codes, these aren't actually being returned by the server right? 

I don't have a better solution for this but this should at least be documented I think